puppet/modules/slave/manifests/init.pp

```diff
@@ -114,6 +114,12 @@
       name => $osfamily ? {
         Debian => "libaugeas-dev",
         default => "augeas-devel"
+      };
+    "lib-qt":
+      ensure => present,
+      name => $osfamily ? {
+        Debian => "libqt4-dev",
+        default => "libqt4-devel"
```

Comment:
"capybara-webkit requires Qt version 4.8 or greater" according to the project's README too, so I don't think that EL6 satisfies this, it appears to have 4.6.